cpg-concepts/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/concepts/memory/cxx/CXXDynamicLoadingPass.kt

```diff
@@ -97,10 +97,15 @@ class CXXDynamicLoadingPass(ctx: TranslationContext) : ConceptPass(ctx) {
         val path =
             call.arguments.getOrNull(0)?.followPrevDFG {
                 it is CallExpression && it.operationNodes.any { it is LoadLibrary }
-            }
+            } // Todo Propagate assumptions
```

Comment:
I have no idea what I need to do here? Do I, as a someone who writes a pass, need to propagate the assumptions? That would not be something that I expect

Reply:
Removed todo comment for now